docs/cli.md

```diff
@@ -0,0 +1,82 @@
+---
+id: cli
+title: CLI
+---
+
+# Key Import using signatory-client
+
+To import a secret key, we will use the `signatory-cli` command.
+
+## Generating a key using tezos-client
+
+This is not the only way to generate keys to import in signatory. Any existing key can be imported in the vaults via signatory if the vault supports the key Curve.
+
+```bash
+% tezos-client gen keys import-p256 -s p256 --encrypted
+Enter password to encrypt your key:
+Confirm password:
+```
+
+```bash
+% tezos-client list known addresses
+import-p256: tz3gxd1y7FdVJ81vzvuACcVjAc4ewXARQkLo (encrypted sk known)
+```
+
+The encrypted private key can be obtained from the `.tezos-client/` directory
+
+```bash
+% cat ~/.tezos-client/secret_keys
+[ { "name": "import-p256",
+    "value":
+      "encrypted:p2esk**********************************************************" }]
+```
+
+## Importing the generated key using signatory-cli
+
+A private key can be imported into any of the backend vaults (except: AWS & ledger) using the below command.
+If you import an encrypted key, `signatory-cli` command will prompt you for the password.
```

Comment:
"If you import an encrypted key, `signatory-cli` command will prompt you for the password."  
->
"If you import an encrypted key, the `signatory-cli` command will prompt you for the password."